docs/_layouts/default.html

```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name">{{ site.url | remove:'https://' }} Statistics</h1>
+				<h1 class="project-name"> {{ site.url | remove:"https://" }} Statistics</h1>
```

Comment:
Sorry for being ambiguous. I meant that a space should be inserted after `remove:` like this: `remove: "https://"` (and not in the beginning. Could you change that?

I know I’m a bit pedantic, but I want to keep the coding style consistent as much as possible 😃.